app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'danikfox1616@gmail.com'
```

Comment:
this is out of current task scope but this is good starting point to meet and understand how to serve different values in different enviroments.  There are also few approaches 
1) Use https://github.com/bkeepers/dotenv
2) https://edgeguides.rubyonrails.org/security.html#custom-credentials
Personally I prefer point 1)

Reply:
'dotenv-rails' is not available in 'asdf: 2.6.3'
second version also don't work  - command not found